src/main/java/com/example/springboottask/service/UserServiceImpl.java

```diff
@@ -52,8 +62,8 @@ public User getUser(Long userId) {
 //                    }
 //                }
 //        );
-
-        return userRepository.findById(userId);
+        UserEntity userEntity = userRepository.findById(userId).get();
```

Comment:
Same here. Don't use Optional.get(). Try to use Optional correctly, like this

    @Override
    public User getUser(Long userId) {
        return userRepository.findById(userId)
                .map(userEntityConverter::convertToModel)
                .orElseThrow(...);
    }